ingest/Ingest.py

```diff
@@ -0,0 +1,102 @@
+__author__ = 'Hao'
+
+import multiprocessing
+from SPARQLWrapper import SPARQLWrapper, JSON
+import itertools
+import json
+import requests
+
+class Ingest:
+    """say something"""
+
+    def __init__(self, object_index, object_type,
+                 get_objects_query_location, describe_object_query_location, variable_name_sparql):
+        self.object_index = object_index
+        self.object_type = object_type
+        self.records = []
+        self.get_objects_query = Ingest.load_file(get_objects_query_location)
+        self.describe_object_query = Ingest.load_file(describe_object_query_location)
+        self.variable_name_sparql = variable_name_sparql
+
+
+    def load_file(filepath):
```

Comment:
So I am now trying to add some neccesary comments.. Could I just use the normal way of adding some comments, instead of using docstring or pydoc?